docs/content/Historical-Config.md

```diff
@@ -100,3 +100,13 @@ You can optionally only configure caching to be enabled on the historical by set
 |`druid.cache.hosts`|Command separated list of Memcached hosts `<host:port>`.|none|
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
+
+
+### Namespaces
+Namespaces, or query time lookups, refers to the ability to modify a datasource with a simple mapping from one set of dimension values to another. The namespaces are a property of dimension extraction functions and can be used in any place dimension extraction functions can be used.
+The following are settings used by the historical nodes when setting namespaces
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.query.extraction.namespace.cache.type`|Specifies the type of caching to be used by the namespaces. May be one of [`offHeap`, `onHeap`, `cache`]. `offHeap` uses a temporary file for off-heap storage of the namespace. `onHeap` stores all cache on the heap. `cache` attempts to piggyback the setup for `druid.cache`|`onHeap`|
```

Comment:
In such a case I can see this changing how the namespace object itself is viewed. Each namespace would then have an identifier (its name) and an updateSpec which included information to the cluster on how and when to get data.

The updateSpec would contain information on if each node should cache a result and how it would update its local cache, or if the cache is some cluster-wide implementation that is queried per key (memcached for example) and how those updates should be handled.

Does that fit better into the cluster-wide view?